src/shared/components/person/profile.tsx

```diff
@@ -383,11 +414,31 @@ export class Profile extends Component<any, ProfileState> {
                     {i18n.t("send_secure_message")}
                   </a>
                   <Link
-                    className={"d-flex align-self-start btn btn-secondary"}
+                    className={"d-flex align-self-start btn btn-secondary mr-2"}
                     to={`/create_private_message/recipient/${pv.person.id}`}
                   >
                     {i18n.t("send_message")}
                   </Link>
+                  {!this.isBlocked() && (
```

Comment:
Use a ternary like the other examples in this codebase. `this.isBlocked ? () : ()`